builtin/providers/aws/hosted_zones_test.go

```diff
@@ -0,0 +1,19 @@
+package aws
+
+import (
+	"testing"
+)
+
+func TestHostedZoneIDForRegion(t *testing.T) {
+	if r := HostedZoneIDForRegion("us-east-1"); r != "Z3AQBSTGFYJSTF" {
```

Comment:
It was slightly more complex previously because it normalized `""` to `"us-east-1"`, but yes a little overkill now.